src/nidigital/examples/nidigital_burst_with_programmatic_configuration.py

```diff
@@ -0,0 +1,182 @@
+    configure_voltage.add_argument('--voh', default=1.7, type=float, help='The output voltage above which the comparator on the pin driver interprets a logic high (H)')
+    configure_voltage.add_argument('--vterm', default=2, type=float, help='The termination voltage the instrument applies during non-drive cycles when the termination mode is set to Vterm')
+    configure_voltage.add_argument('-term-mode', '--termination-mode', default='High_Z', choices=['High_Z', 'Active_Load', 'Three_Level_Drive'])
+    configure_voltage.add_argument('--iol', default=0.002, type=float, help='The maximum current that the DUT sinks while outputting a voltage below VCOM')
+    configure_voltage.add_argument('--ioh', default=-0.002, type=float, help='The maximum current that the DUT sources while outputting a voltage above VCOM')
+    configure_voltage.add_argument('--vcom', default=0.0, type=float, help='The commutating voltage level at which the active load circuit switches between sourcing current and sinking current')
+
+    configure_time_set = subparser.add_parser('configure-time-set', help='Configure time set of the digital pattern based on command line arguments')
+    configure_time_set.add_argument('-c', '--channels', default='PinGroup1', help='Channel(s)/Pin(s) to configure')
+    configure_time_set.add_argument('--period', default=0.00000002, type=float, help='Period in second')
+    configure_time_set.add_argument('-format', '--drive-format', default='NR', choices=['NR', 'RL', 'RH', 'SBC'], help='Non-return | Return to low | Return to high | Surround by complement')
+    configure_time_set.add_argument('--drive-on', default=0, type=float, help='The delay in seconds from the beginning of the vector period for turning on the pin driver')
+    configure_time_set.add_argument('--drive-data', default=0, type=float, help='The delay in seconds from the beginning of the vector period until the pattern data is driven to the pattern value')
+    configure_time_set.add_argument('--drive-return', default=0.000000015, type=float, help='The delay in seconds from the beginning of the vector period until the pin changes from the pattern data to the return value, as specified in the format.')
+    configure_time_set.add_argument('--drive-off', default=0.00000002, type=float, help='The delay in seconds from the beginning of the vector period to turn off the pin driver when the next vector period uses a non-drive symbol (L, H, X, V, M, E).')
+    configure_time_set.add_argument('--strobe-edge', default=0.00000001, type=float, help='The time in second when the comparison happens within a vector period')
+
+    args = parser.parse_args(argsv)
+
+    if args.command == 'configure-voltage':
+        voltage_config = VoltageLevelsAndTerminationConfig(args.vil, args.vih, args.vol, args.voh, args.vterm, args.termination_mode, args.iol, args.ioh, args.vcom)
```

Comment:
I combinate both User Story 1225884: Recreate "Configure Voltage Levels and Termination" LV example in Python and User Story 1225885: Recreate "Create Time Sets" LV example in Python into the same example because they are basically the same thing.

Reply:
That makes sense. But you might want to rename the example to highlight the main thing it's demonstrating - configuring the time set and voltage levels.

How about "nidigital_configure_time_set_and_voltage_levels"?

Reply:
Renamed everything to "configure_time_set_and_voltage_levels"